onnxruntime/core/optimizer/compute_optimizer/upstream_gather_actors.cc

```diff
@@ -462,6 +462,27 @@ bool LayerNormalizationGatherActor::PreCheck(const Graph& /* graph */,
   return true;
 }
 
+bool LayerNormalizationGatherActor::PostProcess(Graph& /*graph*/, Node& current_node,
+                                                const SliceInfo& info_without_node,
+                                                const logging::Logger& /*logger*/,
+                                                const std::unordered_map<int, int>& /*propagate_input_indices*/,
+                                                const std::unordered_map<int, std::vector<DimCompare>>&
+                                                /*all_input_cmp_rets*/,
+                                                const std::unordered_map<int, SliceInfo>& /*new_gather_infos*/) {
+  // Update LayerNormalization's axis attribute if it is scalar slice.
+  if (info_without_node.is_scalar_slice) {
```

Comment:
what does info_without_node mean?

Reply:
There is an explanation for it here https://github.com/microsoft/onnxruntime/blob/86238fb5071ec668cbd0084992e3f408f4ab23fa/onnxruntime/core/optimizer/compute_optimizer/upstream_gather_actors.h#L144C23-L144C23. 

The flow is like this:

slice_info is created initially containing the initial Gather node pointer, 


PreCheck use the info to check whether it chould pass through 
Then upstream the node, remove the Gather node from graph
PostProcess get the info, but the Gather node already removed, so we should not operate on it any more at this point, that's the reason to give a name like this.